avocado/utils/pci.py

```diff
@@ -280,6 +280,24 @@ def get_pci_id(pci_address):
         return ":".join(pci_id)
 
 
+def get_pci_info(pci_address):
+    """
+    Gets PCI info of given PCI address.
+    :param pci_address: Any segment of a PCI address (1f, 0000:00:1f, ...)
+    :return: Dictionary attribute name as key and attribute value as value.
+    :rtype: Dict
+    """
```

Comment:
@maramsmurthy maintain proper line of space in docstring.

Reply:
Sure Naresh! will add line spaces as suggested

Reply:
Addressed suugested changes